core/templates/dev/head/pages/exploration_player/progress_nav_directive.html

```diff
@@ -36,3 +36,19 @@
     </continue-button>
   </div>
 </div>
+
+<style>
+  /*
+    This affects the progress nav bar.
+  */
+  .progress_nav_bar {
```

Comment:
Please prefix this (and other occurrences) with the directive name so that it doesn't apply too generally.

Reply:
Done!